src/Shared/LayoutRenderers/AspNetRequestRouteParametersLayoutRenderer.cs

```diff
@@ -22,7 +22,7 @@ namespace NLog.Web.LayoutRenderers
     /// </code>
     /// </example>
     [LayoutRenderer("aspnet-request-routeparameters")]
-    public class AspNetRequestRouteParametersRenderer : AspNetLayoutMultiValueRendererBase
+    public class AspNetRequestRouteParametersLayoutRenderer : AspNetLayoutMultiValueRendererBase
```

Comment:
Though it seems tempting, then you can repair past mistakes with major-version-bump (Ex. NLog.Web v6)

Please revert, then we can make an issue for v6 about fixing this.